administration/src/util/translations/de.json

```
@@ -335,7 +335,12 @@
     "freinetAgencyId": "AgenturId",
     "freinetAgencyAccessKey": "Zugangsschlüssel",
     "freinetActivateDataTransferCheckbox": "Personen- und Kartendaten an Freinet übermitteln",
-    "freinetActivateDataTransferSuccessful": "Ihre Einstellungen zur Datenübermittlung wurden erfolgreich gespeichert."
+    "freinetActivateDataTransferSuccessful": "Ihre Einstellungen zur Datenübermittlung wurden erfolgreich gespeichert.",
+    "applicationConfirmationMailNoteHeadline": "Eingangsbestätigung Antrag",
+    "applicationConfirmationMailNoteDialogPlaceholder": "Fügen Sie hier Ihre Notiz ein...",
+    "applicationConfirmationMailNoteExplanation": "Hier können Sie eine allgemeine Notiz hinterlegen, welche dem Antragssteller in der Eingangsbestätigungsmail des Antrags zusätzlich angezeigt wird. Sie können die Notiz jederzeit aktivieren oder deaktivieren.",
+    "applicationConfirmationMailNoteActivated": "Notiz bei Antragsbestätigung an Antragssteller:in senden",
```

```suggestion
    "applicationConfirmationMailNoteActivated": "Notiz aktivieren",
```
maybe this would be more straightforward.
otherwise I'm not sure, maybe it's not clear - if I don't check the box, does that mean the note won't be sent to Antragssteller or not sent at all?

---

this note is only for the "Antragssteller" so if not activated it won't be attached to the mail.

---

i think your suggestion should be good

---

I understand that, I mean the current label might be ambitious (maybe just a personal feeling, up to you)